### ISSUE TITLE
npm run build fails at inliner

### ISSUE THREAD

**Issue by cossssmin** [foundation/foundation-emails#538]
To begin with, I work behind a corporate proxy.

I have multiple Foundations for Emails projects, but I'll be talking about one for work and a personal one. The issue is with the personal one, the work-related one works perfectly fine.

For the personal project `npm start` works fine, but when I need to inline and `npm run build`, it produces this:

![image](https://cloud.githubusercontent.com/assets/1656595/17137692/2a1ade7a-5345-11e6-9de1-f548fdc2e87c.png)
#### What I've tried:
- `npm update` - same error
- remove `node_modules` and re-install with `npm install` - same error
- I even copied the `package.json` from the project that works fine, then did both the above - error

Googling it brings up various advice about forcing npm to not use SSL, and so on. Thing is, other Foundation for Emails projects work just fine with `npm run build`, so it's somehow related to this one.

Any advice?

**Comment by kball** [comment]
Interesting... are you by chance using an external (hosted) stylesheet in your email, perhaps off of a CDN? Or something else that would result in the inliner fetching things over the web?

**Comment by cossssmin** [comment]
Resolved.

I do have a `<link>` to pull in Roboto from Google Fonts, but that was not it.
I think some files in the `src` folder got corrupted or something, while pushing/pulling with Bitbucket.

It didn't work if I deleted everything, reinstalled the project and overwrite the src folder with my backup.
But before I overwrote it, I fired it up and it worked fine. So it was something with the files in the src folder (no coding error on my side, because the below worked).

I resolved it by:
- making a backup of the HTML and _settings.scss
- deleting everything, reinstalling the framework
- editing the _settings.scss and adding back my customized settings. There was no error in my code, I simpy copy/pasted back what I had.

OK, I'm closing and filing this under 'weird things that happen once a year' :)
